src/sema/ast.rs

```diff
@@ -366,26 +366,17 @@ pub struct SolanaAccount {
     pub generated: bool,
 }
 
-#[derive(Debug)]
-pub enum ConstructorAnnotation {
-    Seed(Expression),
-    Payer(pt::Loc, String),
-    Space(Expression),
-    Bump(Expression),
+#[derive(Debug, Default)]
+pub struct ConstructorAnnotations {
+    pub seeds: Vec<Expression>,
```

Comment:
no location information for seeds?

Reply:
I use the location information to detect duplicate annotations. This is the case for space and bump. There can be multiple seeds, on the other hand, so I need not track their locations. Shall I add them?

Reply:
I think this is useful information to have in the ast.

Reply:
Fixed